src/Row.php

```diff
@@ -66,6 +66,20 @@ public function remove(string $key): void
         unset($this->attributes[$key]);
     }
 
+    /**
+     * Get a row attribute, and remove it.
+     *
+     * @return mixed
+     */
+    public function pull(string $key)
+    {
+        $value = $this->attributes[$key] ?? null;
+
+        unset($this->attributes[$key]);
+
+        return $value;
```

Comment:
Hello, thanks for the PR :)

Here some suggestion, maybe we could use these already existing methods
```suggestion
        $value = $this->get($key);
        $this->remove($key);

        return $value;
```

Reply:
Hi Nicolas, thanks for your prompt reply.

While I understand it would make sense from a do-not-repeat-yourself perspective, I think we should also take performance into consideration. Method calls are relatively expensive and given how simple the implementation is, I believe we should stick with directly accessing and unsetting the values.

Reply:
That make sense, i am OK with that :)

What you think @ecourtial ?

Reply:
Agree :)